public/javascripts/i18n.js

```diff
@@ -12,19 +12,23 @@
 // See tests for specific formatting like numbers and dates.
```

Comment:
I don't actually we need to modify this file at all actually. Feel free to ignore any Codeclimate messages related message here.